Cubical/Data/Fin/Literals.agda

```diff
@@ -0,0 +1,17 @@
+{-# OPTIONS --cubical --no-import-sorts --no-exact-split --safe #-}
+module Cubical.Data.Fin.Literals where
+
+open import Cubical.Data.Nat.Literals public
```

Comment:
Does this need to be imported `public` for the literals to work?

Reply:
No. I wasn't careful enough.

Reply:
Removed.